src/ImageSharp/ColorSpaces/Companding/SRgbCompanding.cs

```diff
@@ -167,7 +167,7 @@ public static float Compress(float channel)
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     private static unsafe void CompandAvx2(Span<Vector4> vectors, float[] table)
     {
-        fixed (float* tablePointer = &table[0])
+        fixed (float* tablePointer = &MemoryMarshal.GetArrayDataReference(table))
```

Comment:
This avoids the bound-checks.
See [sharplab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAMIR8AB14AbGFADKMgG68wMXAG4a9Jq049+ggJI8ZEMfKhKV6mpoDMDbrmwAzGE1IMhDGgG8aDAKZ7YhQGAFlyAApgAE8MGABtAF0GABNsDGwASm9qQIY/PPzA514EGFSGaLiYACoGMQwoBgBeNIzs/2KAwu7ugDEICEjGqCyNIu6AXy6Amcnc/OJg0LDSavjk9syc2YK9/NLyyo26hqbWhgAydMyEuiSsg8DevvzB4dHx57m9+fy9ssmKtbKctrdsosXj8GEcKlVYvF6qNLlcwjB8NAYmFsFBcAALbCSFgAcRgGAAglAoNgYgARDpsGCuWBcFSRCFZJ4LfKvN4BD4jJrfHmBf5ivZ7BLojD4iCpAziSSRGVyhVKgDyjV4EC4uBYADkIEZJLwuGaAOZZJJ7MRQXgKDJuEIMQWIs5fPY+Pp/GhTIA===) for example.